spec/koine/event_manager/event_listener_spec.rb

```diff
@@ -0,0 +1,66 @@
+require "spec_helper"
+
+class SayHello
+  attr_reader :output, :name
+
+  def initialize(output, name)
+    @output = output
+    @name = name
+  end
+end
+
+class SayGoodBye < SayHello; end
+
+
+describe Koine::EventManager::EventListener do
+  subject { Koine::EventManager::EventListener.new }
+
+  describe "#listen_to" do
+    it "adds listeners"do
+      subject.listen_to(SayHello) do
+      end
+
+      subject.listen_to('SayHello') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.